clients/include/utility.hpp

```diff
@@ -6043,6 +6043,35 @@ class Arguments
     }
 };
 
+inline void missing_file_error_message(const char* filename)
+{
+    std::cerr << "#" << std::endl;
+    std::cerr << "# error:" << std::endl;
+    std::cerr << "# cannot open file '" << filename << "'" << std::endl;
+    std::cerr << "#" << std::endl;
+    std::cerr << "# PLEASE READ CAREFULLY !" << std::endl;
+    std::cerr << "#" << std::endl;
+    std::cerr << "# What could be the reason of this error: " << std::endl;
+    std::cerr << "# You are running the testing application then it expects to find the file "
```

Comment:
application and it expects

or you did not specify the location

Reply:
matrices, or you did not

Reply:
Sorry that first one should be:

...application, then it expects...

Reply:
"You are running the testing application and it expects"